verification/extendTCI.go

```diff
@@ -35,17 +35,25 @@ func TestExtendTCI(d TestDPEInstance, c DPEClient, t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	lastCumulative := tcbInfo.Fwids[1].Digest
+	lastCumulativeTCI := tcbInfo.Fwids[1].Digest
 
 	// Set current TCI value
 	_, err = c.ExtendTCI(handle, tciValue)
 	if err != nil {
 		t.Fatalf("[FATAL]: Could not extend TCI: %v", err)
 	}
 
-	// Check current and cumulative measurement by CertifyKey
-	expectedCumulative := computeExpectedCumulative(lastCumulative, tciValue)
-	verifyMeasurements(c, t, handle, tciValue, expectedCumulative)
+	// Refresh TCB info
+	_, tcbInfo, err = getTcbInfoForHandle(c, handle)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	// Check current and cumulative measurement in DiceTcb info block
+	wantCurrentTCI := tciValue
+	if err = verifyDiceTcbDigest(tcbInfo, wantCurrentTCI, lastCumulativeTCI); err != nil {
+		t.Errorf("[ERROR]: %v", err)
```

Comment:
Just moved the function to certs.go as it is a utility function and shared by other test cases for TCB validation